### PR TITLE
Aligning fidesops docs with changes to fidesctl docs

### PR DESCRIPTION
**Pull request by suchiroll**
# Purpose

Aligning fidesops css with fidesctl css

# Changes

brand colors, copy adjustments, see https://github.com/ethyca/fides/pull/410 and https://github.com/ethyca/fides/pull/436 for more details. 

image

-
# Checklist

- [ ] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md))
- [ ] Good unit test/integration test coverage
- [ ] This PR contains a DB migration. If checked, the reviewer should confirm with the author that the [down_revision correctly references the previous migration](https://ethyca.github.io/fidesops/development/contributing_details/#alembic-migrations) before merging
- [ ] The `Run Unsafe PR Checks` label has been applied, and checks have passed, if this PR touches any external services

# Ticket

Fixes #

### CODE REVIEW

**Review by conceptualshark — APPROVED** [review]
👍 Looks great!

File: docs/fidesops/docs/css/fides.css [hunk 1]
```diff
@@ -18,9 +18,16 @@
     --md-code-hl-string-color: rgb(133, 217, 150);
     --md-code-hl-keyword-color: rgb(245, 251, 255);
     --md-code-hl-punctuation-color: rgb(245, 251, 255);
-    --md-bg-footer-color: #0861CE;
+    --md-code-hl-number-color:  #F8B886;
+    --md-code-hl-special-color: #F8B886;
+    --md-bg-footer-color: #111439;
     --md-bg-table-header: rgba(79, 86, 107, 0.05);
     --svg-highlight-col: #0657b9;
+    --code-window-bg:  #2A2F45;
```
Comment (eastandwestwind) [inline docs/fidesops/docs/css/fides.css:26]: just curious, is there any reasoning behind using hex in some places vs rgb in others?

Comment (suchiroll) [reply docs/fidesops/docs/css/fides.css:26]: When I started working on this there was already a mix of rgb and hex used, I tried to stick with it whatever format it was already done in when adjusting the values, but defaulted to hex when adding new values as it's what we use for the brand site/guidelines and its easier for me to recognize. I can go back and re-do the values in rgba if that's preferred!

Comment (eastandwestwind) [reply docs/fidesops/docs/css/fides.css:26]: no worries, what you did is fine for this PR!

**Pull request merged**